articles/concepts/toc.yml

```diff
@@ -16,3 +16,7 @@
   href: circuits.md
 - name: Quantum oracles
   href: oracles.md
+- name: Further reading
+  href: for-more-info.md
+- name: Glossary
+  href: Glossary.md
```

Comment:
If you're moving the file anyway, it's a good opportunity to change it to be lower-case for cleaner-looking URLs.